config/deploy/qa.rb

```diff
@@ -2,7 +2,7 @@
 
 set :rails_env, :production
 set :bundle_without, %w[development test].join(' ')
-set :branch, 'qa'
+set :branch, 'updateruby'
```

Comment:
It is incorrect to change the branch on QA.  That should remain "qa".  When you make a test deploy of your branch, you can change the branch name LOCALLY on your branch, but that change should not be pushed up to QA.

Reply:
I changed this back to QA i thought on latest update